lib/Compilation.js

```diff
@@ -840,7 +840,9 @@ class Compilation extends Tapable {
 			const module = preparedEntrypoint.module;
 			const name = preparedEntrypoint.name;
 			const chunk = this.addChunk(name);
-			const entrypoint = new Entrypoint(name);
+			const entrypoint = this.namedChunkGroups.get("name")
```

Comment:
This might be bad for error handling, as if the entrypoint resolves to nothing, then it could be a wrong configuration. Do you have a repro?